tests/parameters.h

```diff
@@ -36,9 +57,10 @@ struct NonNegativeParameter : public Parameter<double, NonNegativeParameter> {
 
 struct PositiveParameter : public Parameter<double, PositiveParameter> {
     static inline constexpr double _min = DBL_TRUE_MIN;
-    static inline const std::string _desc = "Non-negative, potentially infinite parameter";
+    static inline const std::string _desc = "Non-negative, non-zero, potentially infinite parameter";
```

Comment:
Seems a bit unnecessarily verbose...
```suggestion
    static inline const std::string _desc = "Positive, potentially infinite parameter";
```